blog/_posts/2021-08-20-Mitchell-Johnson_Jamie-Schmidt.md

```diff
@@ -0,0 +1,296 @@
+  
+In short, our work has greatly simplified the end user experience and streamlined the research process for all who would use MABE2 in the future. 
+
+## Future plans / Work we didn't complete
+ 
+Unfortunately, we simply did not have the time to do everything we wanted.  As such, these are the remaining issues:
+  
+  1. The config system for Brains and Genomes does not yet work.  The structure should be similar to how Organisms are, with each Genome having a name and the various pertinent values (such as mutation rate and min/max values) and each Brain having a name, the names of the Genomes it will use, and whatever stats it will have (such as Markov Brains needing the size of their gates and all Brains having the name of their outputs).  
+    - This is the desired form of DynamicOrg and is really the key to making it as seamless as possible. It allows most of the users work to be done inside the config file instead of in C++.
+  2. Brains should be implemented to actually use this information.
+    - Brains should have their own parameters and settings that appear in the config file.
+  3. BasicBrains, MarkovBrains, and TypedGenomes should all be put in their own files.
+    - For organizational purposes.
+  4. The MarkovBrain should be properly tested and the hardcoded values should be made easily configurable via the config file.
+    - Similar to issue 1, it should work in the config file.
+  5. As part of item 4, GenomeManager and BrainManager classes should be created to act as OrganismManagers do for Genomes and Brains.  Ideally, these would all be turned into one big Manager class.
+    - This is how issue 1 will be achieved, through the implementation of a generic manager class so that essentially any type can be placed into the config system.
+  6. The more complicated mutation types should be added to TypedGenome with configuration options to set the rate for each of them.
+    - Genomes should be highly customizable to allow highly custom experiments to be run. We want to keep the user out of the C++ code as much as possible.
+  7. More Brain types should be added by default.
+    - We currently only have BasicBrains and MarkovBrains, more should be included so that users have to do as little work as possible.
+  
+## Acknowledgements
+
+Mentors: Dr. Charles Ofria & Cliff Bohm
+Participants: Mitchell Johnson & Jamie Schmidt
+
+Charles Ofria and Cliff Bohm conceived and planned the idea of DynamicOrg. They both then contributed to the design and implementation of DynamicOrg. The majority of implementation of DynamicOrg was created by Jamie Schmidt with help from Mitchell Johnson.
+
+Jamie Schmidt wrote the sections on Brains, Genomes, Evaluators, and Introduction. Mitchell Johnson wrote the sections on Tutorial using DynamicOrg, and Conclusion. Jamie Schmidt and Mitchell Johnson worked together on sections Future plans, and DynamicOrg.
+
+Thank you to our amazing mentors!
```

Comment:
I think there's a boilerplate disclaimer that needs to go at the end of this post 
"This work is supported through Active LENS: Learning Evolution and the Nature of Science using Evolution in Action (NSF IUSE #1432563). Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation."

Reply:
Good post overall! Very thorough explanation of what you did this summer! The code snippets were very helpful for understanding how things were implemented. It would be nice if you could all emojis, bolding, or italisizing to emphasize different parts of this post. Right now it is hard to tell what some of the main points are and that adding that formatting in would help!

Reply:
Thank you for the helpful suggestions!